Agriculture_Estate/AgricultureEstate.csproj

```diff
@@ -28,6 +28,9 @@
     </ItemGroup>
 
     <ItemGroup>
+      <Reference Include="Bannerlord.BLSE.LauncherEx">
```

Comment:
Why is that needed?

Reply:
my bad, it's a mistake. will be fixed later. please wait a few minutes